server/webapp/WEB-INF/rails.new/app/assets/javascripts/environment_variables.js

```diff
@@ -138,8 +137,12 @@ EnvironmentVariables = function() {
         });
     };
 
-    function createWrapperElement(tag_type) {
-        return jQuery(document.createElement(tag_type));
+    function createWrapperElement(tag_type, template_text) {
+        if(tag_type){
+          return jQuery(document.createElement(tag_type)).html(template_text);
+        } else {
+          return jQuery(template_text);
```

Comment:
Not sure if I understand this well. Did u really mean to return jQuery(template_text). Template text would normally be some html and not a type or a selector.